exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers[numbers.length] = 4;
```

Comment:
Good job using the more general solution `numbers[numbers.length]` instead of `numbers[3]`